AnnoyancesFilter/sections/cookies_specific.txt

```diff
@@ -1,3 +1,10 @@
+!
+! This section contains the list of the rules that block cookie notifications. Rules should be specific.
```

Comment:
`This section contains the list of the rules` => `This section contains the list of [domain]specific rules`